src/asvz_bot.py

```diff
@@ -408,46 +426,64 @@ def __get_enrollment_and_start_time(driver):
-        )
-        # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
-        enrollment_start_raw = (
-            enrollment_interval_raw.get_attribute("innerHTML")
-            .split("-")[0]
-            .split(",")[1]
-            .strip()
-        )
         try:
-            enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
-        except ValueError as e:
-            logging.error(e)
-            raise AsvzBotException(
-                "Failed to parse enrollment start time: '{}'".format(
-                    enrollment_start_raw
-                )
+            enrollment_interval_raw = driver.find_element(
+                By.XPATH, "//dl[contains(., 'Einschreibezeitraum')]/dd"
+            )
+            # enrollment_interval_raw is like 'So, 09.05.2021 06:35 - Mo, 10.05.2021 07:05'
+            enrollment_start_raw = (
+                enrollment_interval_raw.get_attribute("innerHTML")
+                .split("-")[0]
+                .split(",")[1]
+                .strip()
             )
+            try:
+                enrollment_start = datetime.strptime(enrollment_start_raw, "%d.%m.%Y %H:%M")
+            except ValueError as e:
+                logging.error(e)
+                raise AsvzBotException(
+                    "Failed to parse enrollment start time: '{}'".format(
+                        enrollment_start_raw
+                    )
+                )
+        except NoSuchElementException as e:
+            logging.info("No enrollment time found. Assuming that enrollment is open.")
+            enrollment_start = datetime.today()
         return enrollment_start
 
     @staticmethod
     def __get_lesson_time(driver):
-        lesson_interval_raw = driver.find_element(
-            By.XPATH, "//dl[contains(., 'Datum/Zeit')]/dd"
-        )
-        # lesson_interval_raw is like 'Mo, 10.05.2021 06:55 - 08:05'
-        lesson_start_raw = (
-            lesson_interval_raw.get_attribute("innerHTML")
-            .split("-")[0]
-            .split(",")[1]
-            .strip()
-        )
+        try:
+            lesson_interval_raw = driver.find_element(
+                By.XPATH, "//dl[contains(., 'Datum/Zeit')]/dd"
+            )
+            # lesson_interval_raw is like 'Mo, 10.05.2021 06:55 - 08:05'
+            lesson_start_raw = (
+                lesson_interval_raw.get_attribute("innerHTML")
+                .split("-")[0]
+                .split(",")[1]
+                .strip()
+            )
+        except NoSuchElementException:
+            # If no section called "Datum/Zeit" is found, look for "dt" with "Lektionen"
+            lesson_interval_raw = driver.find_element(
+                By.XPATH, "//dt[contains(., 'Lektionen')]/following-sibling::dd[1]"
+            )
+            # Assuming the "dd" element's innerHTML is '10.05.2021 06:55 - 08:05', and there are multiple "dd" elements
```

Comment:
The format I am seeing is `Fr, 01.12.2023 07:15 - 08:30`, but it shouldn't matter as the code is already handling this format.